python/pysrc/kaskada/_timestream.py

```diff
@@ -1013,6 +1013,126 @@ def count(self, window: Optional[kd.windows.Window] = None) -> Timestream:
         """
         return _aggregation("count", self, window)
 
+    def count_if(self, window: Optional[kd.windows.Window] = None) -> Timestream:
+        """
+        Create a Timestream containing the count value if true in the `window`.
```

Comment:
"count of `true` values in `window`"?

(eg., "count value if true" could suggest that it is `.count().if_(v)`, rather than counting the `true` values).